src/include/kdump/uifunctions.rb

```diff
@@ -946,8 +948,8 @@ def StoreTargetKdumpHandle(type)
         else
           Ops.set(@KDUMP_SAVE_TARGET, "dir", "")
         end
-      elsif radiobut == "ssh"
-        Ops.set(@KDUMP_SAVE_TARGET, "target", "ssh")
+      elsif %w[ssh sftp].include?(radiobut)
+        Ops.set(@KDUMP_SAVE_TARGET, "target", radiobut)
```

Comment:
Nitpicking: What's the difference between `radiobut` and `radiobutton`? This seems to be relic from the past, and should be consistent